src/js/vic-v2/helpers.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash/fp';
+
+export function prefillTransform(pages, formData, metadata) {
+  if (formData && Array.isArray(formData.serviceBranches) && formData.serviceBranches.length) {
+    const newData = _.unset('serviceBranches', formData);
+    newData.serviceBranch = formData.serviceBranches[0];
+    return {
+      metadata,
+      formData: newData,
+      pages: _.set('veteranInformation.schema.properties.serviceBranch.enum', formData.serviceBranches, pages)
```

Comment:
Also, does this restrict the applicant's choice of branch? Would it make sense to simply list those at the top of the full list, rather than restrict the options in case the pre-filled data is incomplete (is this a consideration?) and the the preferred branch is not listed?

Reply:
It does, it only allows you to choose from the ones that we've found in emis. As far as I know, that's what we want to do, but we can change it if we want. @stacyhlinn @lalexanderson-dsva

Reply:
I'm going to go ahead and create a ticket for this question and merge this PR. It seems like a question that requires more thought and research.